scripts/irods/test/test_itree.py

```diff
@@ -66,6 +66,30 @@ def test_pattern_and_ignore_options_together__6791_6806(self):
         self.admin.assert_icommand(["itree",  "--regex-extended", "--ignore-regex", ".*[3-9]", "--pattern-regex", "ba(r|t)[[:digit:]]"],
                                     "STDOUT_MULTILINE", to_catch, use_regex = True)
 
+    def _verify_absolute_object_path(self, logical_path, must_exist):
+        self.assertEqual(logical_path[:1], '/', 'logical_path must start with "/"')
+        if must_exist:
+            self.admin.assert_icommand(['ils',logical_path], 'STDOUT_SINGLELINE', logical_path, desired_rc = 0)
+
+    def test_with_data_object_path__issue_6627(self):
+        PATH_TO_DATA_OBJECT = "data_object_6627"
```

Comment:
Consider dropping the `PATH_TO_` prefix.

It is really just the name of the data object. This also allows you to drop `FULL_` from the other variable name.

Feel free to ignore this comment. Just pointing it out.

_This applies to the other test too._

Reply:
`PATH_TO` has been taken out.